rai/node/node.cpp

```diff
@@ -2739,9 +2758,15 @@ rai::uint128_t rai::election::minimum_threshold (MDB_txn * transaction_a, rai::l
 	return ledger_a.supply (transaction_a) / 16;
 }
 
+rai::uint128_t rai::election::reconfirm_threshold (MDB_txn * transaction_a, rai::ledger & ledger_a)
+{
+	// Minimum number of votes required to consider safe for reconfirmation
+	return ledger_a.supply (transaction_a) / 3;
```

Comment:
Another magic number. Have to be careful with this one, since a block won't be considered reconfirmed unless it meets this threshold. That being said, a block not being reconfirmed has no impact on the node itself, it's purely information for some other system talking to the node over RPC.

Reply:
I think low/medium/high is in the eye of the beholder.  This is a user-visible knob like bitcoin's "# of confirmations" that individuals can customize to meet the needs of their users/transactions.